plugins/modules/utm_aaa_group.py

```diff
@@ -28,7 +28,7 @@
 options:
   name:
     description:
-      - The name of the object. Will be used to identify the entry.
+      - The name of the object identifies the entry.
```

Comment:
I think it's better to keep two sentences here. That the option denotes the object's name is the main thing that is said here, in your version it's only a side-node.

Same in the next few modules.

Reply:
I had added a case in andebox for "will" but the ni umber of occurrences exploded, so I commented that rule out. The first PRs of this batch had it on.

Reply:
Maybe:
```suggestion
      - The name of the object that identifies the entry.
```

Reply:
Or 
```suggestion
      - The name of the object identifying the entry.
```

Reply:
Both are better than the version in this PR, but I still think the two sentence variant is better, especially for non-native speakers.

Reply:
Maybe that's a question for the DaWGs :)

Reply:
Reverted these for now.